src/utils/LaunchDarkly/features.ts

```diff
@@ -53,4 +54,18 @@ const getVariableType = (variations: any[]) => {
     }
 
     return VariableType.json
+}
+
+const mapSDKVisibility = (clientSideAvailability: LDFeature['clientSideAvailability']) => {
+    if (!clientSideAvailability) {
+        return undefined
```

Comment:
this might throw an error if `sdkVisibility` is sending undefined, if there's no options to send it should also not send it as a field in the feature itself and let it use the mongo defaults

Reply:
I think mongoose defaults if the field is undefined so it should be fine

Reply:
nest validation might stop before it gets there